### ISSUE TITLE
Can't sign up with full app

### ISSUE THREAD

**Issue by helpimnotdrowning** [openwanderer/example-app#2]
This is my first time dealing with this database stuff, so sorry if this turns out to be some missed step or dumb mistake.

When I try to sign up using the full app, nothing happens and the Apache `error.log` shows this:
```
[Thu Jun 24 11:06:40.523152 2021] [php7:notice] [pid 8572:tid 1836] [client ::1:50892] Slim Application Error
Type: PDOException
Code: 7
Message: SQLSTATE[08006] [7] could not translate host name &quot;dbname=&quot; to address: Unknown host
File: C:\\Users\\owo\\openwanderer\\full\\vendor\\openwanderer\\openwanderer\\src\\OpenWanderer.php
Line: 30
Trace: #0 C:\\Users\\owo\\openwanderer\\full\\vendor\\openwanderer\\openwanderer\\src\\OpenWanderer.php(30): PDO->__construct('pgsql:host=;dbn...', NULL, NULL)
#1 [internal function]: OpenWanderer\\OpenWanderer::OpenWanderer\\{closure}()
#2 C:\\Users\\owo\\openwanderer\\full\\vendor\\php-di\\invoker\\src\\Invoker.php(74): call_user_func_array(Object(Closure), Array)
#3 C:\\Users\\owo\\openwanderer\\full\\vendor\\php-di\\php-di\\src\\Definition\\Resolver\\FactoryResolver.php(80): Invoker\\Invoker->call(Object(Closure), Array)
#4 C:\\Users\\owo\\openwanderer\\full\\vendor\\php-di\\php-di\\src\\Definition\\Resolver\\ResolverDispatcher.php(71): DI\\Definition\\Resolver\\FactoryResolver->resolve(Object(DI\\Definition\\FactoryDefinition), Array)
#5 C:\\Users\\owo\\openwanderer\\full\\vendor\\php-di\\php-di\\src\\Container.php(387): DI\\Definition\\Resolver\\ResolverDispatcher->resolve(Object(DI\\Definition\\FactoryDefinition), Array)
#6 C:\\Users\\owo\\openwanderer\\full\\vendor\\php-di\\php-di\\src\\Container.php(138): DI\\Container->resolveDefinition(Object(DI\\Definition\\FactoryDefinition))
#7 C:\\Users\\owo\\openwanderer\\full\\vendor\\openwanderer\\openwanderer\\src\\Controllers\\UserController.php(14): DI\\Container->get('db')
#8 C:\\Users\\owo\\openwanderer\\full\\vendor\\php-di\\php-di\\src\\Definition\\Resolver\\ObjectCreator.php(143): OpenWanderer\\Controllers\\UserController->__construct(Object(DI\\Container))
#9 C:\\Users\\owo\\openwanderer\\full\\vendor\\php-di\\php-di\\src\\Definition\\Resolver\\ObjectCreator.php(71): DI\\Definition\\Resolver\\ObjectCreator->createInstance(Object(DI\\Definition\\ObjectDefinition), Array)
#10 C:\\Users\\owo\\openwanderer\\full\\vendor\\php-di\\php-di\\src\\Definition\\Resolver\\ResolverDispatcher.php(71): DI\\Definition\\Resolver\\ObjectCreator->resolve(Object(DI\\Definition\\ObjectDefinition), Array)
#11 C:\\Users\\owo\\openwanderer\\full\\vendor\\php-di\\php-di\\src\\Container.php(387): DI\\Definition\\Resolver\\ResolverDispatcher->resolve(Object(DI\\Definition\\ObjectDefinition), Array)
#12 C:\\Users\\owo\\openwanderer\\full\\vendor\\php-di\\php-di\\src\\Container.php(138): DI\\Container->resolveDefinition(Object(DI\\Definition\\ObjectDefinition))
#13 C:\\Users\\owo\\openwanderer\\full\\vendor\\slim\\slim\\Slim\\CallableResolver.php(146): DI\\Container->get('OpenWanderer\\\\Co...')
#14 C:\\Users\\owo\\openwanderer\\full\\vendor\\slim\\slim\\Slim\\CallableResolver.php(103): Slim\\CallableResolver->resolveSlimNotation('OpenWanderer\\\\Co...')
#15 C:\\Users\\owo\\openwanderer\\full\\vendor\\slim\\slim\\Slim\\CallableResolver.php(72): Slim\\CallableResolver->resolveByPredicate('OpenWanderer\\\\Co...', Array, 'handle')
#16 C:\\Users\\owo\\openwanderer\\full\\vendor\\slim\\slim\\Slim\\Routing\\Route.php(366): Slim\\CallableResolver->resolveRoute('OpenWanderer\\\\Co...')
#17 C:\\Users\\owo\\openwanderer\\full\\vendor\\slim\\slim\\Slim\\MiddlewareDispatcher.php(81): Slim\\Routing\\Route->handle(Object(Slim\\Http\\ServerRequest))
#18 C:\\Users\\owo\\openwanderer\\full\\vendor\\slim\\slim\\Slim\\MiddlewareDispatcher.php(81): Slim\\MiddlewareDispatcher->handle(Object(Slim\\Http\\ServerRequest))
#19 C:\\Users\\owo\\openwanderer\\full\\vendor\\slim\\slim\\Slim\\Routing\\Route.php(341): Slim\\MiddlewareDispatcher->handle(Object(Slim\\Http\\ServerRequest))
#20 C:\\Users\\owo\\openwanderer\\full\\vendor\\slim\\slim\\Slim\\Routing\\RouteRunner.php(84): Slim\\Routing\\Route->run(Object(Slim\\Http\\ServerRequest))
#21 C:\\Users\\owo\\openwanderer\\full\\vendor\\slim\\slim\\Slim\\Middleware\\RoutingMiddleware.php(59): Slim\\Routing\\RouteRunner->handle(Object(Slim\\Http\\ServerRequest))
#22 C:\\Users\\owo\\openwanderer\\full\\vendor\\slim\\slim\\Slim\\MiddlewareDispatcher.php(147): Slim\\Middleware\\RoutingMiddleware->process(Object(Slim\\Http\\ServerRequest), Object(Slim\\Routing\\RouteRunner))
#23 C:\\Users\\owo\\openwanderer\\full\\vendor\\slim\\slim\\Slim\\Middleware\\ErrorMiddleware.php(107): class@anonymous->handle(Object(Slim\\Http\\ServerRequest))
#24 C:\\Users\\owo\\openwanderer\\full\\vendor\\slim\\slim\\Slim\\MiddlewareDispatcher.php(147): Slim\\Middleware\\ErrorMiddleware->process(Object(Slim\\Http\\ServerRequest), Object(class@anonymous))
#25 C:\\Users\\owo\\openwanderer\\full\\vendor\\slim\\slim\\Slim\\MiddlewareDispatcher.php(81): class@anonymous->handle(Object(Slim\\Http\\ServerRequest))
#26 C:\\Users\\owo\\openwanderer\\full\\vendor\\slim\\slim\\Slim\\App.php(215): Slim\\MiddlewareDispatcher->handle(Object(Slim\\Http\\ServerRequest))
#27 C:\\Users\\owo\\openwanderer\\full\\vendor\\slim\\slim\\Slim\\App.php(199): Slim\\App->handle(Object(Slim\\Http\\ServerRequest))
#28 C:\\Users\\owo\\openwanderer\\full\\index.php(10): Slim\\App->run()
#29 {main}, referer: http://localhost/
```
There also seems to be some interesting lines right above that:
```
[Thu Jun 24 11:19:24.614588 2021] [php7:notice] [pid 8572:tid 1836] [client ::1:50421] PHP Notice:  Undefined index: DB_HOST in C:\\Users\\owo\\openwanderer\\full\\vendor\\openwanderer\\openwanderer\\src\\OpenWanderer.php on line 30, referer: http://localhost/
[Thu Jun 24 11:19:24.615564 2021] [php7:notice] [pid 8572:tid 1836] [client ::1:50421] PHP Notice:  Undefined index: DB_DBASE in C:\\Users\\owo\\openwanderer\\full\\vendor\\openwanderer\\openwanderer\\src\\OpenWanderer.php on line 30, referer: http://localhost/
[Thu Jun 24 11:19:24.615564 2021] [php7:notice] [pid 8572:tid 1836] [client ::1:50421] PHP Notice:  Undefined index: DB_USER in C:\\Users\\owo\\openwanderer\\full\\vendor\\openwanderer\\openwanderer\\src\\OpenWanderer.php on line 30, referer: http://localhost/
[Thu Jun 24 11:19:24.616541 2021] [php7:notice] [pid 8572:tid 1836] [client ::1:50421] PHP Notice:  Undefined index: DB_PASS in C:\\Users\\owo\\openwanderer\\full\\vendor\\openwanderer\\openwanderer\\src\\OpenWanderer.php on line 30, referer: http://localhost/
```

I think I set up the `.env` file up correctly:
```
OTV_UPLOADS=C:\xampp\htdocs\data
MAX_FILE_SIZE=1024
DB_HOST=localhost
DB_USER=gisuser
DB_PASS=***
DB_DBASE=gis
```

Using:
Windows 10 20H2
XAMPP
PHP 7.4.20
PostgreSQL 13.3
PostGIS 3.1.2

**Comment by nickw1** [comment]
Hi @helpimnotdrowning I think I have found the cause of this, and have fixed in v0.1.4 of the OpenWanderer server. If you pull from the `example-app` repo and then re-install the dependencies, you will be using v0.1.4 of the server.

It was an issue in the path to `.env` which prevented it loading.

**Comment by helpimnotdrowning** [comment]
It works now! Thanks for your help.
